src/extensions/filter-control/bootstrap-table-filter-control.js

```diff
@@ -499,7 +499,10 @@
             // which ones are going to be present.
             if (controls.length > 0) {
                 this.filterColumnsPartial = {};
-                $(controls[0]).trigger(controls[0].tagName === 'INPUT' ? 'keyup' : 'change');
+                var element = _.find(controls, function(value) {
```

Comment:
Oh, sorry. That is underscore.js, I will rewrite it in a way that it doesn't need it.